CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Unreleased
+
+* Deprecates `Instabug.start` in favour of the new `Instabug.init` API.
```

Comment:
suggestion: I think we need to elaborate more on the usage of the new API or add a docs reference after updating it.

Reply:
We can do this in before the release when all items related to `Instabug.init` are added to the Changelog, then we will work on grouping and rephrasing them together in one solid point.